external-resolver/src/main/scala/org/genivi/sota/resolver/Boot.scala

```diff
@@ -88,6 +89,14 @@ object Boot extends App with Directives with BootMigrations {
     settings.deviceRegistryUri, settings.deviceRegistryApi, settings.deviceRegistryGroupApi
   )
 
+  val messageBusPublisher: MessageBusPublisher =
+    MessageBus.publisher(system, system.settings.config) match {
+      case Xor.Right(c) => c
+      case Xor.Left(err) =>
+        log.error("Could not initialize message bus client", err)
+        MessageBusPublisher.ignore
+    }
+
```

Comment:
Good catch, I've removed it. I'm not sure how we would test this, since we don't test any other messages.